packages/frontend/src/components/HourCommentBox.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="comment-container">
+    <div class="comment-box">
+      <label for="timeEntrieComment">Kommentar</label>
+      <textarea
+        id="timeEntrieComment"
+        v-model="comment"
+        :disabled="disabled"
+        @input="onChange"
+        @keydown.tab="onClickOutside"
+      ></textarea>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
Small detail. Around the code-base we generally use the `Vue.extend`-syntax. This is also what is generally used inside the Vue documentation.

```typescript
export default Vue.extend({
// ... Your component code here
})
```